src/i18n/en.json

```diff
@@ -211,8 +214,48 @@
       "lower-bound": "Lower bound",
       "upper-bound": "Upper bound",
       "subtype": "Subtype",
-      "lbl-loading": "Loading",
-      "lbl-loading-error": "There was an error while loading."
+      "loading-lbl": "Loading",
+      "loading-error-lbl": "There was an error while loading."
+    },
+
+    "issuance": {
+      "loading-lbl": "Loading",
```

Comment:
I'm not sure if the `-lbl` part gives us any information about where the translation is used. You use them for `<th>`, `<button>` contents, no data messages etc... I suggest not using them at all or replace with more contextual prefixes, like `counterpary-heading`, `cancel-btn`, `no-owned-tokens-msg`